.coveragerc

```diff
@@ -1,6 +1,8 @@
 [run]
-source =
-       incremental
+# List of package names.
+source_pkgs = incremental
+# List of directory names.
+source = tests
```

Comment:
```suggestion
source = tests, examplesetup.py
```

Maybe?

Reply:
I don't think it will work 
> source (multi-string): a list of packages or directories, 

from https://coverage.readthedocs.io/en/6.0b1/config.html#run